patroni/ha.py

```diff
@@ -242,6 +245,11 @@ def process_sync_replication(self):
                     if not self.dcs.write_sync_state(self.state_handler.name, None, index=self.cluster.sync.index):
                         logger.info('Synchronous replication key updated by someone else.')
                         return
+
+                if self.is_synchronous_mode_strict() and picked is None:
+                    picked = 'patroni_dummy_host'
+                    logger.info("No standbys available and synchronous_mode_strict set: blocking writes with dummy hostname"
```

Comment:
Tests are failing because:
 1. Line is longer than 120 characters
 2. Missing ')' at the end